frontend/public/components/utils/kebab.tsx

```diff
@@ -108,40 +108,80 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
   />;
 });
 
-export class Kebab extends DropdownMixin {
+export class Kebab extends React.Component<KebabProps, KebabState> {
```

Comment:
I know you're aware, but this will conflict with #1559. The actions dropdowns on lists and details pages will share the same implementation now. It might be better to migrate the kebab separately so all actions dropdowns are consistent.

Reply:
sounds good! I agree.